docs/fsharp/language-reference/xml-documentation.md

```diff
@@ -14,8 +14,7 @@ The compiler-generated XML file can be distributed alongside your .NET assembly
 show quick information about types or members. Additionally, the XML file can be run through tools
 like [fsdocs](http://fsprojects.github.io/FSharp.Formatting/) to generate API reference websites.
 
-XML documentation comments, like all other comments, are ignored by the compiler, unless the options described below are enabled to check the validity and
-completeness of comments at compile time.
+XML documentation comments are not ignored by the compiler; they are processed to generate XML documentation files if the appropriate settings are enabled. Additionally, the compiler can check the validity and completeness of these comments at compile time when specific options are configured.
```

Comment:
Thanks for continuous improvements to our docs, @shethaadit.

I've read this whole doc page, IMO the best option might be to just replace this paragraph by what's written in the `Compile-time checking` section.

Something like,
```suggestion
By default, XML documentation comments are ignored by the compiler. To change this, set `--warnon:3390`. The compiler will then verify the syntax of the XML and the parameters referred to in <param> and <paramref> tags.
```

What do you think? I believe we should mention this "off by default, enable like this" earlier in the article.